pkg/apis/tenancy/v1alpha1/helper/helper.go

```diff
@@ -26,6 +26,13 @@ import (
 	"github.com/kcp-dev/kcp/pkg/apis/tenancy/v1alpha1"
 )
 
+const (
+	WorkspaceNameLabel = "workspaces.kcp.dev/name"
```

Comment:
would even go a step further: move it to types.go. Labels without the "internal" prefix are API.